experimental/segmenter/README.md

```diff
@@ -55,46 +56,64 @@ let breakpoints: Vec<usize> = segmenter.segment_latin1(b"Hello World").collect()
 assert_eq!(&breakpoints, &[6, 11]);
 ```
 
+### Grapheme Cluster Break
+
+Segment a string:
+
+```rust
+use icu_segmenter::GraphemeClusterBreakIterator;
+
+let breakpoints: Vec<usize> = GraphemeClusterBreakIterator::new("Hello World").collect();
+assert_eq!(&breakpoints, &[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]);
```

Comment:
`suggestion (n-b)`: could you incorporate emoji or some other way to make the result less trivial?

Reply:
Nice suggestion! Add a commit using emoji.